active/tab/tabcontrol.md

```diff
@@ -0,0 +1,252 @@
+
+### Selecting a Tab
+When a Tab has focus, pressing SPACE or ENTER will select that Tab.
+
+### Shortcuts for moving between tabs
+Ctrl+TAB will select the next Tab.
+Ctrl+Shift+TAB will select the previous Tab.
+Tab selection cycles (meaning if the user has selected the last tab and presses Ctrl+TAB, the first tab becomes selected).
+
+Ctrl+1 through 8 will automatically select the corresponding Tab.
+Ctrl+9 will automatically select the last Tab.
+
+### Closing a Tab 
+Hitting CTRL+W or CTRL+F4 will automatically close the selected Tab (assuming it is closable). 
+
+### Keyboard guidance for App Developers
+The above sections outline built-in keyboarding behavior provided by the TabControl. However, there are certain expected keyboard shortcuts that you will be responsible for implementing. 
+* If your Tab control supports adding a new Tab, users expect CTRL+T to open a new tab
+* Consider maintaining a list of recently closed Tabs. If the user presses CTRL+SHIFT+T, they expect recently closed tabs to be reopened.
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the Tab. 
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+### TabControl properties and events
+
+| Property | Type | Description |
+|:-------- |:---- |:----------- |
+| CanCloseTabs | bool | Default value for the item if it doesn't specify a IsClosable value. |
+| CloseButtonOverlay | enum | Describes the behavior of the close button of unselected tabs. Values are {Auto, OnHover, Persistent}. Default is Auto. In WinUI 2.2, Auto maps to Persistent.  |
+| ItemHeaderTemplate | DataTemplate | Default template for the item if no template specified. |
+| SelectedTabWidth | double | The size of the selected tab header. |
+| TabHeader | object | Content to the left of the tab strip. |
+| TabHeaderTemplate | DataTemplate | Template for the Header. |
+| TabFooter | object | Content to the far right of the tab strip. |
+| TabFooterTemplate | DataTemplate | Template for the Footer. |
+| TabActionContent | object | Content immediately to the right of the tabs |
+| TabActionContentTemplate | DataTemplate | Template for the ActionContent. |
+| TabWidthBehavior | enum | Specifies how the tabs should be sized. Values are {Actual, Equal, Compact}. Default is Actual. |
+
+| Event | Description |
+|---|---|
+| TabClosing | Fires when a tab is about to be closed. Can be cancelled to prevent closure. |
+| TabDraggedOutside | Fires when a Tab is dragged outside of the Tab bar. |
+
+### TabItem properties and events
+
+| Property | Type | Description |
+|:-------- |:---- |:----------- |
+| Content | object | The main content that appears in the tab. |
+| Header | object | The content that appears inside the tab itself.  |
```

Comment:
What happens regarding the display of the `Header` and `Icon` in a `TabItem` when there is insufficient space to display everything for the number of open tabs and the size of the containing Window? 
It Header content truncated? (remembering that it might be more that text if supporting custom templates.)
Will it slim down to just Icons?
If partially obscured/truncated or just showing the Icon, will there be a tool tip on mouse over to see the full value?

Reply:
and what about scrollbars?

Reply:
The Tab strip area will be contained within a scrollviewer. The tabs themselves will layout relative to their mode (Actual, Equal, Compact). For example, in the scenario where the tabs are Actual, and the app has too many of them, they will each shrink to content and any remaining tabs will clip behind the scrollviewer's bounds. 

The header content will not be truncated unless the app explicitly sets the width of the tab to be less than the natural width of the content. Similarly, it will only slim down to just the icons in the Compact mode.